namer/namer.cc

```diff
@@ -130,13 +130,18 @@ struct FoundMethod final {
 };
 
 struct Modifier {
-    DefinitionKind kind;
+    enum : u1 {
```

Comment:
```suggestion
    enum class Kind : u1 {
```

https://stackoverflow.com/a/32953854

`enum class` is better in almost every way, because `enum` is not much better than an integer.